tests/integrations/quart/test_quart.py

```diff
@@ -0,0 +1,507 @@
+import pytest
+
+quart = pytest.importorskip("quart")
```

Comment:
you'll have to move this to an `__init__.py` like [the others](https://github.com/pgjones/sentry-python/blob/master/tests/integrations/sanic/__init__.py) to fix the test failures. I think pytest tries to parse the whole file and in older versions, the `async def`'s throw syntax errors.

Reply:
Thanks, fixed.